cypress/integration/computation_menu.spec.ts

```diff
@@ -0,0 +1,226 @@
+                    cy.get("#firstDivisor").should("have.value", value);
+                });
+            });
+        });
+    });
+
+    context("Leveling seat threshold", () => {
+        years.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").then(() => waitForLoad());
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const electionThresholdValues = ["0", "5", "7.5", "10", "15"];
+                electionThresholdValues.forEach((electionThreshold) => {
+                    cy.get("#electionThreshold").clear();
+                    cy.get("#electionThreshold").type(electionThreshold);
+                    cy.get("#electionThreshold").should("have.value", electionThreshold);
+                });
+            });
+        });
+    });
+
+    context("District seat threshold", () => {
+        years.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").then(() => waitForLoad());
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const districtThresholdValues = ["0", "5", "7.5", "10", "15"];
+                districtThresholdValues.forEach((districtThreshold) => {
+                    cy.get("#districtThreshold").clear();
+                    cy.get("#districtThreshold").type(districtThreshold);
+                    cy.get("#districtThreshold").should("have.value", districtThreshold);
+                });
+            });
+        });
+    });
+
+    context("Leveling seats", () => {
+        years.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").then(() => waitForLoad());
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const levelingSeatValues = ["0", "25", "50", "75", "100"];
+                levelingSeatValues.forEach((levelingSeats) => {
+                    cy.get("#levelingSeats").clear();
+                    cy.get("#levelingSeats").type(levelingSeats);
+                    cy.get("#levelingSeats").should("have.value", levelingSeats);
+                });
+            });
+        });
+    });
+
+    context("District seats", () => {
+        yearsWithModernElectionSystem.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").then(() => waitForLoad());
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const districtSeatValues = ["0", "125", "250", "325", "500"];
+                districtSeatValues.forEach((districtSeats) => {
+                    cy.get("#districtSeats").clear();
+                    cy.get("#districtSeats").type(districtSeats);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)